### PR TITLE
Allow event-api component

### PR DESCRIPTION
**Pull request by skiddykong**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file, which follo
 on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to 
 [Semantic Versioning](http://semver.org/).
 
+## This release
```
Comment (purple52) [inline CHANGELOG.md]: Convention is to use [Unreleased] for pending changes.

File: core/src/test/java/uk/gov/justice/services/core/sender/SenderProducerTest.java [hunk 2]
```diff
@@ -6,10 +6,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyZeroInteractions;
 import static org.mockito.Mockito.when;
-import static uk.gov.justice.services.core.annotation.Component.COMMAND_API;
-import static uk.gov.justice.services.core.annotation.Component.COMMAND_CONTROLLER;
-import static uk.gov.justice.services.core.annotation.Component.COMMAND_HANDLER;
-import static uk.gov.justice.services.core.annotation.Component.EVENT_PROCESSOR;
+import static uk.gov.justice.services.core.annotation.Component.*;
```
Comment (purple52) [inline core/src/test/java/uk/gov/justice/services/core/sender/SenderProducerTest.java]: Wildcard import

File: generators/generators-test-utils/src/main/java/uk/gov/justice/services/generators/test/utils/dispatcher/BasicRecordingDispatcher.java [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-import static com.jayway.awaitility.Awaitility.await;
-
 import uk.gov.justice.services.messaging.JsonEnvelope;
 
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.function.Predicate;
 
+import static com.jayway.awaitility.Awaitility.await;
```
Comment (purple52) [inline generators/generators-test-utils/src/main/java/uk/gov/justice/services/generators/test/utils/dispatcher/BasicRecordingDispatcher.java]: THis shouldn't have moved...

File: core/src/test/java/uk/gov/justice/services/core/sender/SenderProducerTest.java (hunk 2 above)
Comment (skiddykong) [inline core/src/test/java/uk/gov/justice/services/core/sender/SenderProducerTest.java]: Updated

File: generators/generators-test-utils/src/main/java/uk/gov/justice/services/generators/test/utils/dispatcher/BasicRecordingDispatcher.java (hunk 3 above)
Comment (skiddykong) [inline generators/generators-test-utils/src/main/java/uk/gov/justice/services/generators/test/utils/dispatcher/BasicRecordingDispatcher.java]: Moved back, but surely static at bottom?

**Pull request merged**